### PR TITLE
[inductor][memory] add signpost event for memory pass

### PR DESCRIPTION
**Pull request by xuanzhang816**
Add logging to scuba table for internal models. 

For verification, I triggered a sample workflow internally and checked the scuba table logging to make sure the `Paramaters` column has the expected loggings, see [here](https://fburl.com/scuba/workflow_signpost/39h7qo9s). 

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

### CODE REVIEW

File: torch/_inductor/memory.py [hunk 1]
```diff
@@ -545,11 +549,22 @@ class PeakMemoryResult:
...
-            peak_memory_diff_methods.append(PeakMemoryResult(order, peak_memory))
-            torch_log.warning("%s peak memory: %d", method.__name__, peak_memory)
+            peak_memory_diff_methods.append(
+                PeakMemoryResult(order, peak_memory, method.__name__)
+            )
+            torch_log.info("%s peak memory: %d", method.__name__, peak_memory)
         except Exception as e:
             torch_log.error("Failed to reorder for %s: %s", method.__name__, e)
 
+    # logging to scuba table (fb internal)
```
Comment (yf225) [inline torch/_inductor/memory.py:559]: maybe we can remove this comment (OSS folks shouldn't need to know about scuba)

**Review by yf225 — APPROVED** [review]

**Comment by yf225** [comment]
thanks! maybe need to mark this PR "ready for review"?

File: torch/_inductor/memory.py (hunk 1 above)
Comment (xuanzhang816) [reply torch/_inductor/memory.py]: removed.

**Pull request closed without merging**